### ISSUE TITLE
Feature request: spinny hammer

### ISSUE THREAD

**Issue by SollyBunny** [ddnet/ddnet#9070]
A toggleable easter egg where hammers are rotated like other weapons
I already have the code for this, just wanting to know if I should make a PR

**Comment by ChillerDragon** [comment]
Make a pr here https://github.com/sjrc6/TaterClient-ddnet/  :)

**Comment by sjrc6** [comment]
> Make a pr here https://github.com/sjrc6/TaterClient-ddnet/ :)

I would accept it

**Comment by AssassinTee** [comment]
@heinrich5991 would you accept this as a "feature" for the 1. of April?

**Comment by SollyBunny** [comment]
I use it because it's another way from eyes and hookline to see direction (also looks cool af)

**Comment by ChillerDragon** [comment]
What is

> "feature" for the 1. of April?

@AssassinTee ?

Only one enabled on that day client side like seasonal events?

**Comment by AssassinTee** [comment]
> What is
> 
> > "feature" for the 1. of April?
> 
> [@AssassinTee](https://github.com/AssassinTee) ?
> 
> Only one enabled on that day client side like seasonal events?

> A toggleable easter egg

Yes, like activate it for everyone on 1. of April as a troll, make it toggable elsewhen. Like this is a game, we can have some fun and developers are already claimed to be too strict sometimes 🔨

**Comment by ChillerDragon** [comment]
I think we made ddnet officially a no fun zone a while ago. #9842 #10788

**Comment by heinrich5991** [comment]
> @heinrich5991 would you accept this as a "feature" for the 1. of April?

Probably breaks gameplay too much for people not used to it.

**Comment by SollyBunny** [comment]
> Probably breaks gameplay too much for people not used to it.

People play with leeks and other more questionable texture packs, I would say muscle memory is enough to trump different textures at least for most gameplay

**Comment by AssassinTee** [comment]
> People play with leeks and other more questionable texture packs, I would say muscle memory is enough to trump different textures at least for most gameplay

I think heinrich was talking about new players, who have not yet build up a muscle memory

**Comment by SollyBunny** [comment]
New players would just be screwed with for the day, and can still turn it off.
